lib/AST/ASTImporter.cpp

```diff
@@ -3378,6 +3400,21 @@ ExpectedDecl ASTNodeImporter::VisitFriendDecl(FriendDecl *D) {
 
   while (ImportedFriend) {
```

Comment:
Probably it would be better to make this a for-loop, then `continue` is better to use.

Reply:
Good idea, thanks.